content/docs/command-reference/commit.md

```diff
@@ -40,14 +40,11 @@ scenarios are further detailed below.
   reproduce the whole pipeline. If you're sure no pipeline results would change,
   use `dvc commit` to force update the `dvc.lock` or `.dvc` files and cache.
 
-- In cases where we have previously executed a stage (either by writing
-  `dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later
-  notice that some of the output files or directories it creates, which are
-  already in the <abbr>workspace</abbr>, are missing from `dvc.yaml` (`outs`
-  field). We can
-  [add missing outputs to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
-  without having to execute it again. Use `dvc commit` to update the `dvc.lock`
-  file and save outputs to the cache.
+- In some cases, we have previously executed a stage, and later notice that some
+  of the files/directories used by the stage as dependencies or created as
+  outputs are missing from `dvc.yaml`. It is possible to
+  [add missing data to an existing stage](/docs/user-guide/how-to/add-deps-or-outs-to-a-stage).
+  Use `dvc commit` to update the `dvc.lock` file and save outputs to the cache.
```

Comment:
It is actually better and good enough! 

I would may try still to change the order though. It is possible to add ... 

but it's fine as is I think.

Reply:
Agree on the order of the ideas here, let me think... ⌛